jobs/extensions/extensions/main.py

```diff
@@ -175,6 +175,31 @@ def get_category_from_soup(webpage_soup):
     return category
 
 
+def get_verified_domain(soup):
+    """Input: soup
+    Output: verified_domain string if found, else None"""
```

Comment:
Perhaps it would be better if we followed an existing structure for a docstring to be consistent, here's an example:
https://github.com/mozilla/bigquery-etl/blob/b454a3ba01d57a69597fa3602b5a459478c7134e/bigquery_etl/view/broken_views.py#L50-L60